### ISSUE TITLE
Filebrowser DB Location

### ISSUE THREAD

**Issue by djarbz** [coder/modules#76]
I am looking for a way to modify where the filebrowser.db is stored.
Currently, it looks like it is saving in my working dir.

I would like it to save in `~/.config/...` but it is saving as `/workspace/repo/filebrowser.db`, which is my `coder_agent.dir`.

I think this should be the relevant configuration from my template for this issue.
```
locals {
  folder_name = try(trimsuffix(element(split("/", data.coder_parameter.repo.value), length(split("/", data.coder_parameter.repo.value)) - 1), ".git"), "")
  repo_owner_name = try(element(split("/", data.coder_parameter.repo.value), length(split("/", data.coder_parameter.repo.value)) - 2), "")
  repo_host = try(one(regex("^(?:https?:\\/\\/)?(?:[^@\\/\\n]+@)?(?:www\\.)?([^:\\/\\n]+)", data.coder_parameter.repo.value)), "")
  root_dir = "/workspace"
  project_dir = "${local.root_dir}/${local.folder_name}"
}

resource "coder_agent" "main" {
  arch                   = data.coder_provisioner.me.arch
  os                     = data.coder_provisioner.me.os
  dir                    = local.project_dir
  startup_script_timeout = 180
}

# A file browser for your workspace
module "filebrowser" {
  source = "https://registry.coder.com/modules/filebrowser"
  agent_id = coder_agent.main.id
  folder = local.root_dir
}
```

**Comment by stirby** [comment]
I'll take a look into this and see if we can parameterize the `filebrowser.db` save location.

**Comment by matifali** [comment]
Fixed in #78
